xds/internal/balancer/clustermanager/balancerstateaggregator.go

```diff
@@ -68,34 +65,24 @@ func newBalancerStateAggregator(cc balancer.ClientConn, logger *grpclog.PrefixLo
 	return &balancerStateAggregator{
 		cc:              cc,
 		logger:          logger,
+		csEvltr:         &balancer.ConnectivityStateEvaluator{},
 		idToPickerState: make(map[string]*subBalancerState),
 	}
 }
 
-// Start starts the aggregator. It can be called after Close to restart the
-// aggretator.
-func (bsa *balancerStateAggregator) start() {
```

Comment:
IIUC this pattern was intentional to allow "restarting".  Is that not needed anymore?

Reply:
I think this is a relic from the past where the balancergroup was part of the priority balancer (where restarting is still required). The `clustermanager` being the top-level LB policy is not going to be restarted at any point.